score/README.md

```diff
@@ -1,49 +1,65 @@
 
-## Setup
+## Running using Docker
 
-- Start a terminal
-- Make sure you have Python 3.9 installed: `python -V` or `python3 -V`
-- Create a `virtualenv` in this folder: `python -m venv venv`
-- Activate the virtualenv
-  - Windows: `./venv/Scripts/activate`
-  - Mac/Linux: `source venv/bin/activate`
-- Install packages: `pip install -r requirements.txt`
-  - If you are a Windows user, you might need to install Build Tools for Visual Studio. [Instructions here](https://stackoverflow.com/a/54136652)
+We use Docker to install the necessary libraries in a container that can be run in any operating system.
 
-## Running the Jupyter notebook
+To build the docker container the first time, make sure you're in the root directory of the repository and run `docker-compose build`
 
-- Start a terminal
-- Change to this directory (i.e. `cd score`)
-- Activate your virtualenv (see above)
-- Type `jupyter notebook`. Your browser should open with a Jupyter Notebook tab
+After the container is built, you can start the container wtih `docker-compose up`.
 
-## Activating variable-enabled Markdown for Jupyter notebooks
+To run commands, open a new terminal window or tab:
 
-- Change to this directory (i.e. `cd score`)
-- Run `jupyter contrib nbextension install --user`
-- Run `jupyter nbextension enable python-markdown/main`
-- Make sure you've loaded the Jupyter notebook in a "Trusted" state. (See button near
-  top right of Notebook screen.)
+- Get help: `docker exec j40_score_1 python3 application.py --help`
+- Clean up the data directories: `docker exec j40_score_1 python3 application.py data-cleanup`
 
-For more information, see [nbextensions docs](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html) and
-see [python-markdown docs](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/python-markdown).
+## Local development
+
+You can run the Python code locally to develop, using Poetry. However, to generate the census data you will need the [GDAL library](https://github.com/OSGeo/gdal) installed locally. Also to generate tiles for a local map, you will need [Mapbox tippeanoe](https://github.com/mapbox/tippecanoe)
+
+- Start a terminal
+- Make sure you have Python 3.9 installed: `python -V` or `python3 -V`
+- We use [Poetry](https://python-poetry.org/). Please follow the instructions to download to your system
+- Change to this directory (i.e. `cd score`)
+- Start a Poetry shell with `poetry shell`
 
 ## Downloading Census Block Groups GeoJSON and Generating CBG CSVs
 
 - Make sure you have Docker running in your machine
 - Start a terminal
 - Change to this directory (i.e. `cd score`)
-- Activate your virtualenv (see above)
-- Run `python scripts/download_cbg.py`
+- If you want to clear out all data and tiles from all directories, you can run: `poetry run python application.py data-cleanup`.
+- Then run `poetry run python application.py census-data-download`
```

Comment:
When I try to run this, I get an error: `ModuleNotFoundError: No module named 'dynaconf'` -- I think we need to add `poetry install` to the steps, either here or in the section above.

Reply:
This is fixed i a subsequent commit, hadn't updated `reuirements.txt` from Poetry.